cogs5e/models/sheet/base.py

```diff
@@ -28,27 +28,40 @@ def to_dict(self):
     @classmethod
     def default(cls):
         return cls(0, 10, 10, 10, 10, 10, 10)
+    
+    @classmethod
+    def custstats(cls,cust,origstats):
+        stats=[]
+        abils=('pro',)+STAT_ABBREVIATIONS
+        for i in range(len(abils)):
+            stat=abils[i]
```

Comment:
```suggestion
        for stat in ('pro',)+STAT_ABBREVIATIONS:
```
Since this list of abbreviations is iterated over multiple times, it's probably better to define a new constant `STAT_ABBREVIATIONS_WITH_PROF`